Replaced roman numeral regular expression to address precedence issue

Issue 1: roman number II interpreted as I
https://bitbucket.org/kcartmell/marlowe/issue/1/

--- include/roman_numbers.metaflex.orig	2001-10-18 12:34:31.000000000 +0000
+++ include/roman_numbers.metaflex
@@ -1,5 +1 @@
-ROMAN_THOUSANDS m+
-ROMAN_HUNDREDS  (c(d|m)|dc{0,3}|c{1,3})
-ROMAN_TENS      (x(l|c)|lx{0,3}|x{1,3})
-ROMAN_ONES      (i(v|x)|vi{0,3}|i{1,3})
-ROMAN_NUMBER    {ROMAN_ONES}|{ROMAN_TENS}{ROMAN_ONES}?|{ROMAN_HUNDREDS}{ROMAN_TENS}?{ROMAN_ONES}?|{ROMAN_THOUSANDS}{ROMAN_HUNDREDS}?{ROMAN_TENS}?{ROMAN_ONES}?
+ROMAN_NUMBER    M{0,4}(CM|CD|D?C{0,3})(XC|XL|L?X{0,3})(IX|IV|V?I{0,3})
